cmake/module/FindBerkeleyDB.cmake

```diff
@@ -2,86 +2,136 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-if(CMAKE_HOST_APPLE)
-  execute_process(
-    COMMAND brew --prefix berkeley-db@4
-    OUTPUT_VARIABLE bdb4_brew_prefix
-    ERROR_QUIET
-    OUTPUT_STRIP_TRAILING_WHITESPACE
-  )
+#[=======================================================================[
+FindBerkeleyDB
+--------------
+
+Finds the Berkeley DB headers and library.
+
+Imported Targets
+^^^^^^^^^^^^^^^^
+
+This module provides imported target ``BerkeleyDB::BerkeleyDB``, if
+Berkeley DB has been found.
+
+Result Variables
+^^^^^^^^^^^^^^^^
+
+This module defines the following variables:
+
+``BerkeleyDB_FOUND``
+  "True" if Berkeley DB found.
+
+``BerkeleyDB_VERSION``
+  The MAJOR.MINOR version of Berkeley DB found.
+
+#]=======================================================================]
+
+if(BREW_COMMAND)
+  # The Homebrew package manager installs the berkeley-db* packages as
+  # "keg-only", which means they are not symlinked into the default prefix.
+  # To find such a package, the find_path() and find_library() commands
+  # need additional path hints that are computed by Homebrew itself.
+  set(_BerkeleyDB_homebrew_include_hint)
+  foreach(_suffix IN ITEMS db@4 db@5 db)
```

Comment:
Why is this checking for multiple (unsupported) versions, when on master we only check for `berkeley-db@4`?

Reply:
Linux users can build using BDB v5. Why should we limit macOS users?

Reply:
Thanks! `db` removed, and the commit message adjusted.

Reply:
I think `db5` should also be removed, unless there's a compelling reason to increase the scope of BDB support here.

Reply:
Done.